### ISSUE TITLE
Sia-Nextcloud crashes siad, loses files

### ISSUE THREAD

**Issue by mtlynch** [NebulousLabs/Sia-Nextcloud#14]
In testing Sia-Nextcloud, I found that it causes siad to crash frequently and lose files.

### To repro

1. Follow [my tutorial](https://mtlynch.io/sia-nextcloud/) for setting up Sia-Nextcloud
1. Log into Nextcloud home screen
1. Upload several files at once to the Sia folder.

### Expected

Files upload successfully to both Sia and Nextcloud.

### Actual

Files upload successfully to Nextcloud, but siad frequently crashes in the middle of uploading to Sia. Sometimes on resume, the files are lost in Sia.

The crashes seem to happen even when I upload one file at a time, but they're more frequent when I upload many files at once.

### Environment:

* Docker on Windows 10
* Sia 1.3.0 (on a Debian Jessie [Docker image](https://hub.docker.com/_/debian/))
* Nextcloud 12.0 ([Docker image](https://hub.docker.com/_/nextcloud/))
* Chrome 60

### Logs

Couldn't find anything relevant in the logs. `renter.log` just has:

```text

2017/08/06 19:01:03.808548 persist.go:403: STARTUP: Logging has started.
2017/08/06 19:03:29.206187 persist.go:403: STARTUP: Logging has started.
2017/08/06 19:04:33.877656 persist.go:403: STARTUP: Logging has started.
2017/08/06 19:05:31.041545 persist.go:403: STARTUP: Logging has started.
```

### Video repro

* https://youtu.be/80mu1v3vcfY

In the video, you can see siad frequently crash. Also notice that at [4:19](https://youtu.be/80mu1v3vcfY?t=4m19s), you see it uploading the file "30 Rock - s01e03 - Blind Date.mp4" to Sia. Sia then crashes and when I restart it, at [4:38](https://youtu.be/80mu1v3vcfY?t=4m38s), the file is gone.

**Comment by avahowell** [comment]
Under this setup, does the crash only occur when Sia is used with nextcloud? Can you verify if uploading via `siac` causes similar behavior?

**Comment by mtlynch** [comment]
Yes, I can repro with just siac. Should I move this to [NebulousLabs/Sia](https://github.com/NebulousLabs/Sia)?

**Comment by avahowell** [comment]
Yeah, that would be ideal. Are you able to monitor `stdout` and `stderr` for the `siad` process when this happens?

**Comment by mtlynch** [comment]
Nothing prints to `stdout` or `stderr`. I tried to repro on Linux+Docker, but I can't repro there, so I feel like this might be specific to Win10+Docker.

I tried to repro the whole Linux+Docker+Sia+Nextcloud setup on Ubuntu 16.04 LTS and in that setup, the Nextcloud instance won't connect to siad at all even though wallet's unlocked and funded, blockchain is sync'ed, contracts are set up. I'll try filing a separate bug on that.

Closing this one because I suspect the chances of getting someone to investigate a bug that happens only on Docker+Win10 is going to be pretty slim.
